apps/daemon/pkg/toolbox/toolbox.go

```diff
@@ -110,7 +129,8 @@ func (s *Server) Start() error {
 		sessionGroup := processController.Group("/session")
 		{
 			sessionGroup.GET("", sessionController.ListSessions)
-			sessionGroup.POST("", sessionController.CreateSession)
+			sessionGroup.POST("", sessionController.CreateSession) // deprecated
+			sessionGroup.POST("/v2", sessionController.CreateSession)
```

Comment:
Since we have the SDK version header, I suggest we use that to determine how to handle the request.

`/v2` will be very hard to maintain later and also requires an additional endpoint in the API.